clouddriver-ecs/src/test/java/com/netflix/spinnaker/clouddriver/ecs/provider/agent/IamRoleCachingAgentTest.java

```diff
@@ -152,4 +154,39 @@ public void shouldGenerateFreshData() {
           keys.contains(cacheData.getId()));
     }
   }
+
+  @Test
+  public void shouldGetDefaultRegion() {
+    // given
+    String defaultRegionName = Regions.DEFAULT_REGION.getName();
+    when(netflixAmazonCredentials.getRegions())
+        .thenReturn(Collections.singletonList(new AmazonCredentials.AWSRegion("us-east-1", null)));
+
+    // when
+    String actualRegionName = agent.getIamRegion();
+
+    // then
+    assertEquals(
+        "Expected region to equal " + defaultRegionName + ", but got " + actualRegionName,
```

Comment:
I think the assert already fails with an error stating what your expected and actual values are. Might be unnecessary

Reply:
ah, I was going by how the other tests in this file are structured; would prefer to keep them consistent for now